composer/utils/inference.py

```diff
@@ -57,6 +57,29 @@ class ExportFormat(StringEnum):
     ONNX = 'onnx'
 
 
+def _move_sample_input_to_device(sample_input: Optional[Union[torch.Tensor, dict, list, Tuple]],
+                                 device: torch.device) -> Optional[Union[torch.Tensor, dict, list, Tuple]]:
+    """Handle moving sample_input of various types to a device. If possible, avoids creating copies of the input."""
+    output = None
+    if isinstance(sample_input, torch.Tensor):
+        output = sample_input.to(device)
```

Comment:
Nit: to be consistent with the rest of `Composer`, can we create `DeviceCPU` class and instead use `DeviceCPU.to(tensor_to_device)`?